relay-server/src/services/store.rs

```diff
@@ -184,6 +184,14 @@ impl StoreService {
     ) -> Result<(), StoreError> {
         let retention = envelope.retention();
         let event_id = envelope.event_id();
+
+        let use_ingest_feedback_topic = self
+            .global_config
+            .current()
+            .options
+            .ingest_topic_rollout_rate
+            > 0.0;
```

Comment:
That's not a rollout rate, it's a boolean. There is `crate::utils::is_rolled_out` if you want a rollout rate, otherwise please change everything to a boolean enabled/disabled option.

Reply:
Thanks! I wasn't sure how to use the rollout rate yesterday